### PR TITLE
fix: `MinWidth` and `MinHeight` should not be applied by default on `TextBox`

### PR DESCRIPTION
**Pull request by MartinZikmund**
GitHub Issue (If applicable): part of https://github.com/unoplatform/uno/issues/9063

## PR Type

What kind of change does this PR introduce?

- Bugfix

## What is the current behavior?

Our generic.xaml applies `MinWidth` and `MinHeight` in the default style

## What is the new behavior?

As per WinUI sources, `MinWidth` and `MinHeight` should only be applied to the border inside of the control, not as properties on the control directly. This way, setting 0 width and height should make the `TextBox` invisible. On the other hand, the default fluent text box style should apply these properties directly.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [x] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [x] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [x] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

### CODE REVIEW

File: src/Uno.UI/UI/Xaml/Style/Generic/Generic.xaml [hunk 1]
```diff
@@ -221,10 +221,6 @@
 
 	<Style x:Key="XamlDefaultTextBox"
 		   TargetType="TextBox">
-		<Setter Property="MinWidth"
```
Comment (Youssef1313) [inline src/Uno.UI/UI/Xaml/Style/Generic/Generic.xaml:224]: In `controls/dev/CommonStyles/TextBox_themeresources.xaml` from WinUI code, I see:

```xaml
<Setter Property="MinHeight" Value="{ThemeResource TextControlThemeMinHeight}" />
<Setter Property="MinWidth" Value="{ThemeResource TextControlThemeMinWidth}" />
```

Comment (Youssef1313) [reply src/Uno.UI/UI/Xaml/Style/Generic/Generic.xaml:224]: Maybe there is something about `ComboBoxTextBoxStyle` not being used properly or the style itself having an issue?

Comment (Youssef1313) [reply src/Uno.UI/UI/Xaml/Style/Generic/Generic.xaml:224]: Reading the tests, I see the intent is that this is only for old UWP styles. If that's the intent, I think it's right and the comment can be resolved

Comment (MartinZikmund) [reply src/Uno.UI/UI/Xaml/Style/Generic/Generic.xaml:224]: yes indeed, the fluent styles have the properties explicit, but the ye old generic "UWP" styles don't 😂 . It is quite odd!

**Review by Youssef1313 — APPROVED** [review]

**Pull request merged**